app/code/Magento/Sales/Controller/Adminhtml/Invoice/AbstractInvoice/View.php

```diff
@@ -1,17 +1,19 @@
 <?php
 /**
- *
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
```

Comment:
Please add `declare(strict_types=1)` to all changed files

> All new PHP files MUST have strict type mode enabled by starting with declare(strict_types=1);. All updated PHP files SHOULD have strict type mode enabled.

See [Magento Technical guideline](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#1-basic-programming-principles) 1.3.1 topic

Reply:
Done, thank you.